CMakeLists.txt

```diff
@@ -181,7 +175,7 @@ if(QUIC_TLS STREQUAL "openssl")
         OUTPUT ${QUIC_BUILD_DIR}/openssl/include
         OUTPUT ${QUIC_BUILD_DIR}/openssl/lib/libcrypto${CMAKE_STATIC_LIBRARY_SUFFIX}
         OUTPUT ${QUIC_BUILD_DIR}/openssl/lib/libssl${CMAKE_STATIC_LIBRARY_SUFFIX}
-        COMMAND ./Configure linux-x86_64 enable-tls1_3 --prefix=${QUIC_BUILD_DIR}/openssl
+        COMMAND SYSTEM=`uname -s` ./config enable-tls1_3 --prefix=${QUIC_BUILD_DIR}/openssl
```

Comment:
Should we always be doing `SYSTEM='uname -s'` or just for Azure Pipelines?

Reply:
In this case, its fine to do it in the command. We'd have to do a LOT more work to make cross compilation work, which is what that is normally used for.